Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -7466,6 +7466,195 @@ script:
       - contextPath: PANOS.DownloadStatus.Summary.started
         description: Whether the download process started.
         type: String
+  - arguments:
+    - default: false
+      description: ID of host (serial or hostname) to reboot
+      isArray: false
+      name: hostid
+      required: true
+      secret: false
+    deprecated: false
+    description: >
+      Reboot the given device by hostid. Warning; this command has no confirmation and the device
+      will immediately reboot.
+    name: pan-os-platform-reboot
+    outputs:
+    - contextPath: PANOS.RestartStatus.Summary.hostid
+      type: String
+      description: Host ID
+    - contextPath: PANOS.RestartStatus.Summary.started
+      description: Whether download process has started.
+      type: String
+  - arguments:
+    - default: false
+      description: ID of host (serial or hostname) to check.
+      isArray: false
+      name: hostid
+      required: true
+      secret: false
+    deprecated: false
+    description: Checks the status of the given device, checking whether it's up or
+      down and the operational mode normal
+    name: pan-os-platform-get-system-status
+    outputs:
+    - contextPath: PANOS.SystemStatus.hostid
+      type: String
+      description: Host ID
+    - contextPath: PANOS.SystemStatus.up
+      description: Whether the host device is up or still unavailable.
+      type: String
+  - arguments:
+    - default: false
+      description: ID of host (serial or hostname) to update the state.
+      isArray: false
+      name: hostid
+      required: true
+      secret: false
+    - default: false
+      description: New state.
+      isArray: false
+      name: state
+      required: true
+      secret: false
+    deprecated: false
+    description: Checks the status of the given device, checking whether it's up or
+      down and the operational mode normal
+    name: pan-os-platform-update-ha-state
+    outputs:
+    - contextPath: PANOS.HAStateUpdate.hostid
+      type: String
+      description: Host ID
+    - contextPath: PANOS.HAStateUpdate.state
+      description: New HA State
+      type: String
+  - arguments:
+    - default: false
+      defaultValue: null
```

Comment:
what's the difference between these two?

Reply:
I removed the `defaultvalue: Null` values as they do nothing. However, my understanding is as follows;

`default: true` flags that the argument has a default value (`true`) implemented in code 
`defaultValue: True` passes the value (`True`) into the argument by default unless it is overridden.

Reply:
further, i think the reason you would use the default flag over defaultValue is if the argument is inherited from integration settings but overrideable, whereas you would use defaultValue if the argument simply has a default you want the user to see in the command.